app/mailers/spree/user_mailer.rb

```diff
@@ -1,7 +1,7 @@
 module Spree
   class UserMailer < BaseMailer
     def reset_password_instructions(user, token, *args)
-      @edit_password_reset_url = spree.edit_spree_user_password_url(:reset_password_token => token)
+      @edit_password_reset_url = spree.edit_spree_user_password_url(:reset_password_token => token, :host => Spree::Config.site_url)
```

Comment:
thanks for showing me where spree sets this value. I think this is a good change, even if we don't push it back to Spree since they set this in the MailerSettings class, which won't work for us due to the host value needing to change per request to support multi-tenancy. :+1: